### DISCUSSION TITLE
Help with configuring "subswaggers" in Swagger-Aggregator

### DISCUSSION THREAD

**Discussion by jdloyd** [mia-platform/community#532] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

Version: 1.102.8

### Description

I am trying to get Categories to work in the API portal. I have been following the docs to enable this feature (part of [swagger aggregator: here](https://docs.mia-platform.eu/docs/development_suite/api-console/advanced-section/swagger-aggregator/configuration#subswaggers)) and have not been able to get it to work. Here is what I have:

```
"subswaggers": {
    "/document-generation.json": {
      "name": "DocumentGeneration - Routes",
      "tagName": "DocumentGeneration",
      "expression": "'/doc-gen/' === path"
    },
    "/document-generation-acl.json": {
      "name": "DocumentGenerationACL - Routes",
      "tagName": "DocumentGenerationACL",
      "expression": "'/doc-gen-acl/' === path"
    }
  }

```
I want a category for all the endpoints in DocumentGeneration and in DocumentGenerationACL (separate micro services).  I'm pretty sure I have something wrong with my path statement as the categories show up in the API Portal, but it does not have any endpoints listed. I need a wildcard for the path expression, but don't know how that would work. Other paths I have tried:

```
'/doc-gen/*'
'/doc-gen'
```

Lastly, I am currently running version 1.16.12 of the API Portal and version 3.5.1 of the Swagger-Aggregator. 

### Actual Outcome

I get the categories in the API portal, but there are no routes listed 

### Expected Outcome

I get the categories in the API portal with the expected routes listed

**Comment by davidebianchi** [comment, marked as answer]
Hi @jdloyd! If you try with something like: `path.startsWith('/doc-gen/')`?

**Reply by jdloyd** [reply]
David - that worked - Thanks so much
